vertx-sql-client/src/main/java/io/vertx/sqlclient/Tuple.java

```diff
@@ -468,6 +469,12 @@ default Buffer getBuffer(int pos) {
     if (val instanceof Buffer) {
       return (Buffer) val;
     }
+    if (val instanceof ByteBuf) {
+      return Buffer.buffer((ByteBuf) val);
+    }
+    if (val instanceof byte[]) {
+      return Buffer.buffer((byte[]) val);
+    }
```

Comment:
CC @vietj for this change. Do you think this is OK? From experimenting with the API returning null in these getters causes very confusing behavior, so I think it's good to be as lenient as possible here.